holonote/annotate/display.py

```diff
@@ -303,6 +303,7 @@ def edit_tools(self) -> list[Tool]:
             tools.append(BoxSelectTool(dimensions="width"))
         elif self.region_format == "point-point":
             tools.append("tap")
+        tools.append("doubletap")
```

Comment:
Can this be moved to `panel/app.py`?

Edit: Can see this actually was in the app. Please move it back.